static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -182,40 +172,40 @@ class InviteMembersModal extends DeprecatedAsyncComponent<
       pendingInvites: [...state.pendingInvites, this.inviteTemplate],
     }));
 
-  setEmails(emails: string[], index: number) {
+  setEmails = (emails: string[], index: number) => {
```

Comment:
I made this change so i could pass around the function without losing the `this` reference.